tbx/static_src/sass/components/_motif-heading.scss

```diff
@@ -19,6 +19,10 @@
         @include reduced-motion() {
             animation: none;
         }
+
+        #{$root}--static & {
+            animation: none;
+        }
```

Comment:
In hindsight I probably should have set the animation to a class of it's own so it doesn't need to be undone

Reply:
I think I'll leave it as is for now - if we find we're overriding it all over the place we can refactor